packages/cardano-services/src/Program/services/postgres.ts

```diff
@@ -117,7 +117,7 @@ export const getConnectionConfig = <Suffix extends ConnectionNames>(
 ): Observable<PgConnectionConfig> => {
   const postgresConnectionString = getPostgresOption(suffix, 'postgresConnectionString', options);
   const postgresSslCaFile = getPostgresOption(suffix, 'postgresSslCaFile', options);
-  const ssl = postgresSslCaFile ? { ca: loadSecret(postgresSslCaFile) } : undefined;
+  const ssl = postgresSslCaFile ? { ca: loadSecret(postgresSslCaFile) } : { rejectUnauthorized: false };
```

Comment:
If this option is needed, then we probably want to set it here as well?
```suggestion
  const ssl = postgresSslCaFile ? { ca: loadSecret(postgresSslCaFile, rejectUnauthorized: false) } : { rejectUnauthorized: false };
```